### PR TITLE
Add  content/zh/cloud_computing.md

### PR DESCRIPTION
**Pull request by Submarinee**
Add cloud_computing.md  and content/zh/container.md localization to zh.

### CODE REVIEW

File: content/zh/container.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+所有这些都是为了让单个应用程序运行。此外，操作系统的维护、升级和启动是另一个重要的工作来源。
+
+## 如何帮助
+
+容器共享相同的操作系统及其机器资源，分散了操作系统的资源开销并创建了物理机器的有效使用。
+这种能力之所以成为可能，是因为容器之间的交互通常受到限制。
+这允许更多的应用程序在同一台物理机器上运行。
+
+但是，有一些限制。由于容器共享相同的操作系统，因此可以认为进程不如替代方法安全。 容器还需要对共享资源进行限制。 为了保证资源，
+管理员必须约束和限制内存和 CPU 的使用，以使其他应用程序不会表现不佳。
```
Comment (Jacob953) [inline content/zh/container.md:25]: It would be better to add the end line. 🤗

Comment (Submarinee) [reply content/zh/container.md:25]: Ok~

**Review by hanyuancheung — APPROVED** [review]
lgtm

**Review by Jacob953 — APPROVED** [review]
LGTM.

**Pull request merged**